### PR TITLE
Phobia DLC - Spooky Pack

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request

This PR expands phobias to include more objects and mobs.

- **Supernatural Phobia** now includes more wizard objects, supernatural creatures and species. (zombies, vampires, nightmares, etc.)
- **Clown Phobia** now includes bananium objects and honk related simple mobs.
- All phobias that had any crossover associations with toys, stamps, statues, or megaphones were also added. (for instance the security phobia now checks for security toys and megaphones) 

## Why It's Good For The Game

More stuff to trigger phobias.

## Changelog

:cl:
expansion: Supernatural phobia now includes more wizard objects, spooky mobs, and haunted objects.
expansion: Clown phobia now includes bananium objects and honk related simple mobs.
expansion: Phobias can be triggered from related stamps, toys, statues, and megaphones.
/:cl:

### CODE REVIEW

**Comment by EricZilla** [comment]
I feel like a fear of the supernatural should not include inert bones

File: code/controllers/subsystem/traumas.dm [hunk 1]
```diff
@@ -104,102 +116,119 @@ SUBSYSTEM_DEF(traumas)
...
 			/turf/open/floor/mineral/abductor, /turf/closed/wall/mineral/abductor
 		)),
+		"clowns" = typecacheof(list(/turf/closed/wall/mineral/bananium, /turf/open/floor/mineral/bananium)),
 		"falling" = typecacheof(list(/turf/open/chasm, /turf/open/floor/fakepit, /turf/open/openspace))
 	)
 
-	phobia_species = list("lizards" = typecacheof(list(/datum/species/lizard)),
-		"skeletons" = typecacheof(list(/datum/species/skeleton, /datum/species/plasmaman)),
+	phobia_species = list(
+		"lizards" = typecacheof(list(/datum/species/lizard)),
```
Comment (RaveRadbury) [inline code/controllers/subsystem/traumas.dm:222]: You're adding a lizard phobia and not documenting it, also lizard phobia species-wise is handled by `conspiracies`. Up to this point we've yet to have a species spook more than one phobia type.

**Comment by timothymtorres** [comment]
I didn't realize the skeleton phobia was meant for plasmamen.  That makes sense actually and I will remove those changes.

**Comment by timothymtorres** [comment]
It's ready to go now.

**Pull request merged**